bodhi/server/validators.py

```diff
@@ -710,6 +710,22 @@ def validate_release(request, **kwargs):
                            "Invalid release specified: {}".format(releasename))
 
 
+def validate_release_name(request, **kwargs):
+    """
+    Make sure the referenced release exists.
+
+    Args:
+        request (pyramid.util.Request): The current request.
+        kwargs (dict): The kwargs of the related service definition. Unused.
+    """
+    release = Release.get(request.matchdict.get('name'))
```

Comment:
If you change the parameter name from ```name``` to ```release```, I think you might be able to reuse ```validate_release()``` defined below instead of creating another validator.

Reply:
```validate_release``` accepts parameter from ```request.validated``` and not the URL. That's why created a new validator